packages/components/package.json

```diff
@@ -118,7 +118,6 @@
     "react-swipeable": "^5.5.1",
     "react-tiny-popover": "^4.0.0",
     "react-transition-group": "^4.2.2",
-    "react-window": "^1.8.5",
-    "tt-react-custom-scrollbars": "^4.2.1-tt2"
+    "react-window": "^1.8.5"
```

Comment:
`tt-react-custom-scrollbars` was uninstalled but no `package-lock.json` changes were detected. Could you please uninstall using the `npm uninstall` command?